content/en/integrations/guide/aws-integration-with-terraform.md

```diff
@@ -0,0 +1,198 @@
+  tag: "Blog"
+  text: "Secure your infrastructure in real time with Datadog Cloud Workload Security"
+- link: "https://www.datadoghq.com/blog/announcing-cloud-siem/"
+  tag: "Blog"
+  text: "Announcing Datadog Security Monitoring"
+---
+
+You can use [Terraform][1] to install and manage the AWS integration with Datadog. The sections below detail the requirements, and provide basic templates for creating resources in Datadog.
+
+## Setup Terraform provider
+
+Configure the [Datadog Terraform provider][2] with your Datadog [API and Application keys][3]. These allow the Terraform provider to access your Datadog account and create resources through Datadog APIs.
+
+## Setup Datadog integration IAM role
+
+Configure permissions for the IAM role used by Datadog to query AWS APIs on your behalf. Use [All Permissions][4] to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. If you would like to use Datadog’s Cloud Security Posture Management product, you must also add additional permissions by attaching the AWS [SecurityAudit Policy][5] to the role. 
+
+The script below creates the Datadog IAM policy inside your AWS account. Update these parameters before running the script:
+
+* `YOUR_DD_EXTERNAL_ID`: A unique ID located in your [Datadog AWS Integration tile][6].
+* `AWS_PERMISSIONS_LIST`: The IAM policies needed by Datadog AWS integrations. 
+
+{{< code-block lang="hcl" filename="" disable_copy="false" collapsible="true" >}}
+variable "datadog_aws_integration_external_id" {
+  default = "<YOUR_DD_EXTERNAL_ID>"
+  description = ""
+}
+
+data "aws_iam_policy_document" "datadog_aws_integration_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type = "AWS"
+      identifiers = ["arn:aws:iam::464622532012:root"]
+    }
+    condition {
+      test = "StringEquals"
+      variable = "sts:ExternalId"
+
+      values = [
+        "${var.datadog_aws_integration_external_id}"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "datadog_aws_integration" {
+  statement {
+    actions = [<AWS_PERMISSIONS_LIST>]
+
+    resources = ["*"]
+  }
+}
+
+resource "aws_iam_policy" "datadog_aws_integration" {
+  name = "DatadogAWSIntegrationPolicy"
+  policy = "${data.aws_iam_policy_document.datadog_aws_integration.json}"
+}
+
+resource "aws_iam_role" "datadog_aws_integration" {
+  name = "DatadogAWSIntegrationRole"
+  description = "Role for Datadog AWS Integration"
+  assume_role_policy = "${data.aws_iam_policy_document.datadog_aws_integration_assume_role.json}"
+}
+
+resource "aws_iam_role_policy_attachment" "datadog_aws_integration" {
+  role = "${aws_iam_role.datadog_aws_integration.name}"
+  policy_arn = "${aws_iam_policy.datadog_aws_integration.arn}"
+}
+{{< /code-block >}}
+
+## Create resources
+
+Use templates to create and manage the Datadog - Amazon Web Services integration.
+
+### Datadog - Amazon Web Services integration
```

Comment:
Let's make this the first section and also include the snippet in the main AWS terraform code block with the IAM role so you can feed the external ID from the `datadog_integration_aws` resource into the external ID value of the IAM role.